packages/e2e-tests/fixtures/blocks/core__table__deprecated-1.html

```diff
@@ -1,3 +1,3 @@
-<!-- wp:core/table -->
-<table class="wp-block-table"><thead><tr><th>Version</th><th>Musician</th><th>Date</th></tr></thead><tbody><tr><td><a href="https://wordpress.org/news/2003/05/wordpress-now-available/">.70</a></td><td>No musician chosen.</td><td>May 27, 2003</td></tr><tr><td><a href="https://wordpress.org/news/2004/01/wordpress-10/">1.0</a></td><td>Miles Davis</td><td>January 3, 2004</td></tr><tr><td>Lots of versions skipped, see <a href="https://codex.wordpress.org/WordPress_Versions">the full list</a></td><td>&hellip;</td><td>&hellip;</td></tr><tr><td><a href="https://wordpress.org/news/2015/12/clifford/">4.4</a></td><td>Clifford Brown</td><td>December 8, 2015</td></tr><tr><td><a href="https://wordpress.org/news/2016/04/coleman/">4.5</a></td><td>Coleman Hawkins</td><td>April 12, 2016</td></tr><tr><td><a href="https://wordpress.org/news/2016/08/pepper/">4.6</a></td><td>Pepper Adams</td><td>August 16, 2016</td></tr><tr><td><a href="https://wordpress.org/news/2016/12/vaughan/">4.7</a></td><td>Sarah Vaughan</td><td>December 6, 2016</td></tr></tbody></table>
-<!-- /wp:core/table -->
+<!-- wp:table {"backgroundColor":"subtle-pale-blue"} -->
```

Comment:
What makes us need to change and existing deprecation? Could we leave this deprecation without changes and create a new deprecation specific to the new case we want to test?

Reply:
My understanding is that the ideal order for deprecations is reverse chronological. In keeping with past advice to that effect, I added the deprecation for the colors block support changes to the beginning of the deprecated array.

I regenerated the deprecation fixtures so they still aligned. That is, the first deprecation in the array matches the fixture named `*_deprecated-1.html`.

Deprecations in general feel very counter intuitive. So making the fixtures not align didn't see like an option. Leaving a choice to either:
- Append the new deprecation to the array and only add new fixtures (which went against past recommendations for deprecation order)
- Prepend the new deprecation and regenerate fixtures (the downside being inconvenience when reviewing).

I went with the later as it was a one time cost and got the slight optimisation of the reverse-chronologically order deprecations.